### ISSUE TITLE
Make switch for HVM/PV mode and show configuration

### ISSUE THREAD

**Issue by borzel** [xcp-ng/xenadmin#60]

**Comment by borzel** [comment]
So, there's the first Idea:

![grafik](https://user-images.githubusercontent.com/5782951/42424230-1b42423a-8309-11e8-8c00-51a8b6d7519f.png)

![grafik](https://user-images.githubusercontent.com/5782951/42424244-77578a30-8309-11e8-8332-861e7435f2e3.png)

**Comment by borzel** [comment]
There's the build to test it: https://builds.schulzalex.de/xcp-ng/feature-convert-PV-HVM/

**Comment by cocoon** [comment]
While working on this problem: https://github.com/xcp-ng/xenadmin/issues/129

I have seen this check:

```
In XCP-NG Center there is an additional check:

eli = !hvm && m_template.PV_bootloader == "eliloader";

If using URL it requires the PV_bootloader to be eliloader.
```

So URL would not work with this?
https://github.com/xcp-ng/xenadmin/blob/ac1bcc6b17f3753335057b1649d3fb4085bd0f03/XenAdmin/SettingsPanels/BootOptionsEditPage.cs#L305

So there would be a need for a switch between the two+ PV_bootloader?

**Comment by borzel** [comment]
Final implementation:

![grafik](https://user-images.githubusercontent.com/5782951/52522879-53d42880-2c8b-11e9-9152-e8e64d3e51ae.png)

![grafik](https://user-images.githubusercontent.com/5782951/52522891-6cdcd980-2c8b-11e9-802f-f4b54ebc253a.png)

**Comment by borzel** [comment]
@cocoon I created a new issue for the implemetation for eliloader: #140

**Comment by borzel** [comment]
finally released with 8.0.0
